arch/xtensa/src/common/xtensa_dumpstate.c

```diff
@@ -53,47 +53,165 @@ static uint32_t s_last_regs[XCPTCONTEXT_REGS];
  ****************************************************************************/
 
 /****************************************************************************
- * Name: up_taskdump
+ * Name: xtensa_dump_stak
```

Comment:
```suggestion
 * Name: xtensa_dump_task
```
Typo in the function name.